package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint src",
     "test": "npm run unit && npm run lint",
-    "unit": "bun run mocha test/**/*.test.js --require test/bootstrap.js",
```

Comment:
wasn't running the TS file, lol, actually I am not sure the bun transpiler is properly working yet. Will follow up with that.

Reply:
I tested adding this as a build script which seems to be working: 
```build": "bun build ./src/index.ts --outdir ./build```

It doesn't output the d.ts files though.. I'll see if I can read up a bit on bun..

Reply:
I think you might configure the .d.ts export behaviour in a tsconfig.json, it definately will read one if there is one around.

Reply:
I switched to bun test, it seems to run it properly without config or bootstrap.js